python/doc/examples/functional_modeling/vectorial_functions/plot_functions_outputDim.py

```diff
@@ -0,0 +1,79 @@
+"""
+===========================================
+Increase the output dimension of a function
+===========================================
+"""
+# %%
+#
+# Import the required modules
+
+# %%
+import openturns as ot
+
+# %%
+# Description
+# ===========
+#
+# We want to build a function :math:`f : \mathbb{R}^d \mapsto \mathbb{R}^q` from *q* functions :math:`f_i: \mathbb{R}^d \mapsto \mathbb{R}`.
+#
+# We can do that:
+#
+#     - Case 1: by aggregation  of the functions :math:`f_i`,
```

Comment:
The items are unnecessarily indented:
![image](https://user-images.githubusercontent.com/31351465/230657518-9aea789e-7023-4b53-a0b3-c91c04443115.png)

```suggestion
#  - Case 1: by aggregation  of the functions :math:`f_i`,
```